src/pages/patientView/mutation/PatientViewMutationsDataStore.ts

```diff
@@ -18,7 +19,24 @@ function mutationIdKey(m: Mutation) {
 export default class PatientViewMutationsDataStore extends SimpleGetterLazyMobXTableApplicationDataStore<
     Mutation[]
 > {
-    @observable.ref private mouseOverMutation: Readonly<Mutation> | null = null;
+    @observable tooltipModel: {
```

Comment:
would be preferable to put this kind of UI related state into component rather than datastore

Reply:
I have moved everything to a TimelineWrapperStore. There's no code change anymore from me in PatientViewMutationsDataStore. But the mouseOverMutation in here is part of old functionality and its purpose is that when a mutation is highlighted in the VAF chart to be highlighted in the mutations table too and vice versa.  Same goes for selected mutations.